langchain/package.json

```diff
@@ -450,6 +450,7 @@
   "dependencies": {
     "@anthropic-ai/sdk": "^0.4.3",
     "@dqbd/tiktoken": "^1.0.4",
+    "ansi-styles": "^5.0.0",
```

Comment:
This is a much lighter dependency which should work in every env. It is used by chalk under the hood